webpack.config.ts

```diff
@@ -1,51 +1,61 @@
-import path from 'path';
-
-const commonConfig = {
-    devtool: 'source-map',
-    mode: 'development',
-    module: {
-        rules: [
-            {
-                exclude: /node_modules/,
-                test: /\.tsx?$/,
-                use: 'ts-loader',
-            },
-        ],
-    },
-    resolve: {
-        extensions: ['.tsx', '.ts', '.js'],
-    },
-};
-
-module.exports = [
-    {
-        ...commonConfig,
-        entry: {
-            host: './src/host/mainHost.ts',
-            messaging: './src/host/mainMessaging.ts',
-        },
-        name: 'host',
-        output: {
-            filename: '[name].bundle.js',
-            path: path.resolve(__dirname, 'out/host'),
-        },
-    },
-    {
-        ...commonConfig,
-        entry: {
-            extension: './src/extension.ts',
-        },
-        externals: {
-            vscode: 'commonjs vscode',
-        },
-        name: 'extension',
-        output: {
-            devtoolModuleFilenameTemplate: '../[resource-path]',
-            filename: '[name].js',
-            libraryTarget: 'commonjs2',
-            path: path.resolve(__dirname, 'out'),
-        },
-        stats: 'errors-only', // Bug ws package includes dev-dependencies which webpack will report as warnings
-        target: 'node',
-    },
-];
+import copyPlugin from 'copy-webpack-plugin';

```

Comment:
changes LGTM but do we know why everything is marked as new?

Reply:
Olivia's PR that modified all double quotes to single just landed. Merge conflicts everywhere. I just rebased. Much easier to review